### ISSUE TITLE
Cannot find module

### ISSUE THREAD

**Issue by LordScree** [mattiash/node-multi-tape#1]
I'm probably doing something wrong... new project, etc. but I've installed node, tape and multi-tape... got a simple test that passes if I call `.\node-modules\.bin\tape test/*.js`... but fails with `Error: Cannot find module '{path}\test\*.js'` when using `.\node-modules\.bin\multi-tape test/*.js`.

Full VS Code/PowerShell output:

```
> .\node_modules\.bin\multi-tape test/*.js

#
# test/*.js
#
module.js:549
    throw err;
    ^

Error: Cannot find module 'C:\...\test\*.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
test/*.js exited with error 1
```

Any suggestions would be most welcome!

**Comment by mattiash** [comment]
I have looked at this briefly and I don't think you are doing anything wrong. I have only used multi-tape on Linux and OS X. On these platforms, if you pass in a wildcard string as a parameter to a command, the wildcard will be expanded to the list of files that it matches by the shell and then the command will get the list of files as its arguments. Apparently, Windows does not seem to do that. So for Windows, I guess multi-tape has to do the expansion itself.

Unfortunately, I don't have any Windows machine to test it on, so I can't really help you with implementing this in multi-tape. Take a look at how other commands handle this (maybe look at tape since that seems to work) and see if you can fix it yourself and submit a pull-request with your fix.

**Comment by LordScree** [comment]
@mattiash thanks for the fast response.

I've had a quick look at the way _tape_ handles it and it looks like tape is using [glob](https://www.npmjs.com/package/glob). I'll have a play about later this week and submit a pull-request, but it may add _glob_ as an additional dependency. Do you think that would be okay?

**Comment by mattiash** [comment]
I think a dependency on the glob module would be ok.

**Comment by mattiash** [comment]
Released 1.2.0.
